.gitignore

```diff
@@ -21,4 +21,5 @@ test.log
 .gobincache
 go.work
 go.work.sum
-*~
\ No newline at end of file
+*~
+internal/cmd/build/*.sqlite*
```

Comment:
Is something creating this file? (no comment on the review itself, LGTM)

Reply:
Yeah, when you run the integ tests w/ dev server mode

Reply:
yes